chapters/samplers.txt

```diff
@@ -478,6 +478,51 @@ ifdef::VK_VERSION_1_2[<<features-samplerMirrorClampToEdge,samplerMirrorClampToEd
 endif::VK_VERSION_1_2,VK_KHR_sampler_mirror_clamp_to_edge[]
 --
 
+[open,refpage='VkCompareOp',desc='Comparison operator for depth, stencil, and sampler operations',type='enums']
```

Comment:
While technically first occurence here, I would prefer it below `VkPipelineDepthStencilStateCreateInfo`, which is more mainstream usage of the enum.